src/main/resources/injection.st

```diff
@@ -0,0 +1,16 @@
+import org.springframework.util.AntPathMatcher
```

Comment:
This could be hidden to user by setting import in the GroovyShell on Java side

Reply:
Done